sync.js

```diff
@@ -218,15 +225,13 @@ class Sync extends events.EventEmitter {
 
   replicateNetwork (peer, opts) {
     if (!peer.handshake) {
-      process.nextTick(function () {
-        peer.sync.emit('error', new Error('trying to sync before handshake has occurred'))
+      process.nextTick(() => {
+        this._syncQueue.set(peer.id, peer)
```

Comment:
@noffle please review!
I added this because if you hit replicateNetwork twice in a row, it would throw an error immediately. This gives the consumer a bit more of what they expect

Reply:
Still applicable? I see you removed the sync queue.

Reply:
Still applicable? I see you removed the sync queue.

Reply:
Yeah I removed it!

Reply:
There were some edge cases and it became clear that it wasn't necessary *now* although a nice to have, better to push it out the door